### PR TITLE
test: add test for map `stop` (#8369)

### LINKED ISSUE #8369: Implement test for 'stop' method of Map

### PR DESCRIPTION
**Pull request by victorpatru**
Implementing test for `stop` method of Map.

First time contributor to Leaflet so please let me know if any adjustments are needed!

Closes #8369

### CODE REVIEW

File: spec/suites/map/MapSpec.js [hunk 1]
```diff
@@ -275,6 +275,23 @@ describe("Map", function () {
...
+			expect(map.stop.called).to.not.be.ok();
+		});
+
+		it("stops the execution of the flyTo animation", function () {
+			map.setView([0, 0]);
+			map.stop = sinon.spy();
+			map.flyTo([51.505, -0.09]);
+			map.stop();
+			expect(map.stop.called).to.be.ok();
+		});
```
Comment (Falke-Design) [inline spec/suites/map/MapSpec.js:292]: Please copy this block and stop `panTo`

**Review by Falke-Design — CHANGES_REQUESTED** [review]

File: spec/suites/map/MapSpec.js (hunk 1 above)
Comment (victorpatru) [reply spec/suites/map/MapSpec.js:292]: Just updated my initial commit. Let me know if there is anything else!

File: spec/suites/map/MapSpec.js [hunk 2]
```diff
@@ -275,6 +275,31 @@ describe("Map", function () {
...
+			map.panTo([10, 10], 10);
+			expect(map.stop.called).to.not.be.ok();
+		});
+
+		it("stops the execution of the flyTo animation", function () {
+			map.setView([0, 0]);
+			map.stop = sinon.spy();
+			map.flyTo([51.505, -0.09]);
+			map.stop();
+			expect(map.stop.called).to.be.ok();
```
Comment (Falke-Design) [inline spec/suites/map/MapSpec.js:291]: `calledOnce` should be better for both tests

```suggestion
			expect(map.stop.calledOnce).to.be.ok();
```

**Review by Falke-Design — CHANGES_REQUESTED** [review]

**Review by Falke-Design — APPROVED** [review]
Thank you @victorpatru

**Pull request merged**